zbus_xmlgen/src/main.rs

```diff
@@ -25,10 +25,17 @@ fn usage() {
   zbus-xmlgen <interface.xml>
   zbus-xmlgen --system|--session <service> <object_path>
   zbus-xmlgen --address <address> <service> <object_path>
+  zbus-xmlgen <interface.xml> -o <output-file>
```

Comment:
this doesn't need to be listed separately. `-o` is supported in all cases. So I'd just append `[-o|--output <file_path>]` to the 3 lines above and remove this line.